test/register.ts

```diff
@@ -0,0 +1,88 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { mkdirp } from './helpers';
+import * as svelte from 'svelte/compiler';
+
+type Compile = typeof svelte.compile;
+type CompileOptions = Parameters<Compile>[1];
+
+const extensions = ['.svelte', '.html'];
+let compileOptions = {};
+let compile;
+let folderName = 'normal';
+
+const sveltePath = process.cwd().split('\\').join('/');
+
+function capitalise(name: string) {
+	return name[0].toUpperCase() + name.slice(1);
+}
+
+export function setCompileOptions(_compileOptions: CompileOptions) {
```

Comment:
should these methods be named with the underscore convention like `set_compile_options`?